aws/lambda.go

```diff
@@ -29,6 +29,23 @@ func (l *Lambda) Execute(ctx context.Context, functionName string, payload []byt
 		SetInvocationType(lambda.InvocationTypeRequestResponse).
 		SetLogType(lambda.LogTypeTail)
 
+	return l.invoke(functionName, ctx, input)
+}
+
+// Execute executes the given function with the given payload and qualifier (lambda version) and returns the output
+func (l *Lambda) ExecuteWithQualifier(ctx context.Context, functionName string, functionQualifier string, payload []byte) ([]byte, error) {
+	input := &lambda.InvokeInput{}
+	input.
+		SetPayload(payload).
+		SetFunctionName(functionName).
+		SetQualifier(functionQualifier).
+		SetInvocationType(lambda.InvocationTypeRequestResponse).
+		SetLogType(lambda.LogTypeTail)
```

Comment:
Seems like this option is causing issues for others https://github.com/chanzuckerberg/blessclient/issues/70. I'm considering just removing the `LogType` option here.